### PR TITLE
upgrade bower-install to 1.0

### PR DESCRIPTION
**Pull request by stephenplusplus**
Yay! `bower-install` 1.0 is out! It now supports Sass files, is called `bowerInstall`, as opposed to `bower-install`, and has some other small config property changes. Give it a shot!

### CODE REVIEW

**Comment by sindresorhus** [comment]
Congrats on 1.0!

File: app/templates/Gruntfile.js [hunk 1]
```diff
@@ -357,9 +361,8 @@ module.exports = function (grunt) {
                 dest: '.tmp/styles/',
                 src: '{,*/}*.css'
             }
-        },
+        },<% if (includeModernizr) { %>
 
-<% if (includeModernizr) { %>
         // Generates a custom Modernizr build that includes only the tests you
         // reference in your app
         modernizr: {
```
Comment (stephenplusplus) [inline app/templates/Gruntfile.js:369]: I did a quick format of this file to avoid those ugly multiple line breaks between config blocks.

**Comment by stephenplusplus** [comment]
Thanks! I'll give this a couple more days if anyone wants to test, then I'll merge 'er in if no errors are reported.

File: app/templates/Gruntfile.js (hunk 1 above)
Comment (addyosmani) [inline app/templates/Gruntfile.js:369]: LGTM.

**Comment by addyosmani** [comment]
:+1:

**Pull request merged**